inst/extdata/partypes.csv

```diff
@@ -0,0 +1,99 @@
+parameter_type_short,parameter_type
+delay_incubation_period_inverse,Human delay - incubation period  (inverse parameter)
```

Comment:
In the spirit of making things easier for the user, can we just call it incubation_period_inverse rather than delay_incubation period?